### DISCUSSION TITLE
Update 6.7

### DISCUSSION THREAD

**Discussion by TBog** [TBog/TBLauncher#216] [Release]
* rename QuickList to Dock
* fix icon and name cache not resetting
* all changes from v6.6-dev.*

<hr /><em>This discussion was created from the release <a href='https://github.com/TBog/TBLauncher/releases/tag/v6.7'>Update 6.7</a>.</em>

**Comment by ltguillaume** [comment]
Saw some list update problem:

1. I went to Untagged via Tags menu
2. I added a tag to a shortcut (one created via [WebApps](https://f-droid.org/en/packages/com.tobykurien.webapps/))
3. Went back to widget mode and entered Tags menu again
4. The shortcut is still among the untagged (without the tag being mentioned)
  If I search for the shortcut, it does turn up with the tag underneath it.

**Comment by ltguillaume** [comment]
The broadcast receiver for uninstalled apps seems to work nicely!

**Comment by ltguillaume** [comment]
Copied from https://github.com/TBog/TBLauncher/discussions/214#discussioncomment-1520669
> I just found out that with (one of) the latest dev build(s), the keyboard also shows when you open a tag (or the tags list) when not yet in search mode. I don't think people would want to search by text if they explicitly open a group of apps, because that searches through all apps. I think that before, the keyboard wouldn't open when opening tags from the tags menu, or directly from the dock (when you pin the dock to the widget mode, for example), right?

> It may even be a good idea to explicitly close the keyboard if people click on a tag (or the tags list) from the dock, when already in search mode.

**Reply by TBog** [reply]
I read the issue before the release, but I wanted to have a new version on Playstore to have the crashes fixed.

**Reply by ltguillaume** [reply]
> 
> 
> I read the issue before the release, but I wanted to have a new version on Playstore to have the crashes fixed.

Cool, just wanted to get it over here to be sure that we don't have to keep visiting all previous discussions, but please let me know how you would prefer to handle stuff like this in these discussions (I'm pretty new to it tbh).

**Reply by TBog** [reply]
It would be great if we could keep the issues separated in [here](https://github.com/TBog/TBLauncher/issues) and link to them from the discussions.
